tests/components/homekit_controller/specific_devices/test_aqara_gateway.py

```diff
@@ -39,14 +36,12 @@
     light = entity_registry.async_get("light.aqara_hub_1563")
     assert light.unique_id == "homekit-0000000123456789-65792"
 
-    light_helper = Helper(
-        hass, "light.aqara_hub_1563", pairing, accessories[0], config_entry
-    )
+    light_helper = Helper(hass, "light.aqara_hub_1563", pairing,
+                          accessories[0], config_entry)
     light_state = await light_helper.poll_and_get_state()
     assert light_state.attributes["friendly_name"] == "Aqara Hub-1563"
-    assert light_state.attributes["supported_features"] == (
-        SUPPORT_BRIGHTNESS | SUPPORT_COLOR
-    )
+    assert light_state.attributes["supported_features"] == (SUPPORT_BRIGHTNESS
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)